### DISCUSSION TITLE
Bilder dynamisch einbinden

### DISCUSSION THREAD

**Discussion by devchristiansimon** [neuefische/questions#22] [Web] [answered]
Ich habe eine JSON, in der der Pfad zur Bilddatei liegt.
Allerdings hab ich es so verstanden, dass man bei den JS und JSX Dateien jedes einzelne Bild importieren muss über import.

Wie kann ich das umgehen, damit ich dynamisch Bild-URLs aus meiner JSON Datei darstellen kann?

**Comment by stefankorder** [comment, marked as answer]
Die Images sollten am besten im public-Ordner der App liegen, natürlich gerne gut strukturiert in einem Unterordner. Der Pfad aus der JSON / Datenbank sollte dann auf diesen Ordner verweisen, also z.B. `/images/beispielbild.jpg`.
